lib/components/user/monitored-trip/trip-monitored-days.tsx

```diff
@@ -0,0 +1,70 @@
+import { FormattedList, FormattedMessage, useIntl } from 'react-intl'
+
+import InvisibleA11yLabel from '../../util/invisible-a11y-label'
+import React from 'react'
+
+import { ALL_DAYS } from '../../../util/monitored-trip'
+import FormattedDayOfWeek from '../../util/formatted-day-of-week'
+import FormattedDayOfWeekCompact from '../../util/formatted-day-of-week-compact'
+import getBaseColor from '../../util/base-color'
+
+import styled from 'styled-components'
+
+interface Props {
+  days: string[]
+}
+
+const DayCircleContainer = styled.div`
+  display: flex;
+  gap: 4px;
+`
+
+const MonitoredDay = styled.span<{ baseColor: string; monitored: boolean }>`
+  align-items: center;
+  background-color: ${(props) =>
+    props.monitored ? props.baseColor : 'transparent'};
+  border: 1px solid ${(props) => (props.monitored ? props.baseColor : '#333')};
+  border-radius: 50%;
+  color: ${(props) => (props.monitored ? 'white' : 'inherit')};
+  display: flex;
+  height: 27px;
+  justify-content: center;
+  opacity: ${(props) => (props.monitored ? 1 : 0.7)};
+  text-transform: capitalize;
+  width: 27px;
```

Comment:
Oy vey... No way to avoid this?

Reply:
Should we convert these units to `rem` and then use base height to get these back to the right size? that should maybe help with our spacing issues

Reply:
Converted to rem!